main.py

```diff
@@ -209,15 +211,15 @@ async def run_client():
     async with client:
         await check(client)
         try:
-            await debug(client, "Freegan has started", INFO)
+            # await debug("Freegan has started", INFO)

```

Comment:
why is the start log message is commented out?

Reply:
Oh, it was useless for me while testing and I'm not sure it has to be in the final version. But for now I've resurrected it :)